templates/thank-you.html

```diff
@@ -16,20 +15,62 @@ <h1 data-qa="title">{{ _("Submission successful") }}</h1>
     "classes": "u-mb-s"
     })
   %}
+    <h1 data-qa="title">{{ _("Thank you for submitting the {survey_title}").format(
```

Comment:
I notice in the screenshot attached to the trello card `Thank you for completing the ...` is used here, however the text in the content box on the trello card matches what you've added. Might be worth confirming with @donnap573 which is correct?

Reply:
Discussed with @donnap573 and fixed, correct version is attached to Trello card.